docs/docs/facts/market.md

```diff
@@ -14,20 +14,20 @@ To understand the market size, we flow from the global language services market
 [^marketresearchfuture]: Market Research Future. 2024. [Translation Service Market](https://www.marketresearchfuture.com/reports/translation-service-market-1400).
 [^slator]: Slator. 2022. [2022 Language Industry Market Report](https://slator.com/slator-2022-language-industry-market-report/).
 
-From a more holistic perspective, the entire sign language economy in the U.S. alone is estimated to range between **US\$ 3 billion** and **US\$ 10 billion** annually.
+From a broader perspective, the entire sign language economy in the U.S. alone is estimated to range between **US\$ 3 billion** and **US\$ 10 billion** annually.
 This includes diverse services and sectors that support the deaf and signing community.[^gallaudet]
 
 [^gallaudet]: Gallaudet University. 2022. [How Sign Language is Driving a Multi-Billion Dollar Inclusive Economy](https://gallaudet.edu/president/how-sign-language-is-driving-a-multi-billion-dollar-inclusive-economy/).
 
-To estimate the global sign language market, we assume that similarly developed nations in the Top 30 Human Development Index[^hdi] are mirroring U.S. trends, especially regarding accessiblity regulation and inclusivity.
-Among others, they have enacted the European Accessibility Act[^eu-accessiblity-act], the Accessible Canada Act (2019)[^canada-act], and Australia’s Disability Discrimination Act (1992)[^australia-act].
+To estimate the global sign language market, we extrapulate based on trends in developed nations ranked within the Top 30 Human Development Index[^hdi], which are mirroring U.S., especially regarding accessiblity regulation and inclusivity.
+This include the recently enacted the European Accessibility Act[^eu-accessiblity-act], the Accessible Canada Act (2019)[^canada-act], and Australia’s Disability Discrimination Act (1992)[^australia-act].
```

Comment:
I think that `Among others` better communicates "including but not limited to" than "This include the"
(but in any case, if you disagree, this is also not grammatical. `This includes` would be correct)